src/components/calcite-tile-select/calcite-tile-select.scss

```diff
@@ -14,49 +14,40 @@ $spacing: $baseline/2;
   position: relative;
   transition: $transition;
   vertical-align: top;
+  @apply z-10;
+}
+
+:host(:hover) {
+  @apply z-20;
+  box-shadow: 0 0 0 1px var(--calcite-ui-border-1);
```

Comment:
does this work in RTL?

Reply:
I think box shadow spread would be the same, but looking at the component there aren't any RTL considerations for rendering content or styling. I'll open a separate issue for that. The nomenclature for "show-input" should also be changed to "start" and "end"

<img width="1547" alt="Screen Shot 2020-11-05 at 11 08 36 AM" src="https://user-images.githubusercontent.com/4733155/98285458-6ca06980-1f57-11eb-9a6a-8657d985c7da.png">

Reply:
Follow up issue for RTL support: https://github.com/Esri/calcite-components/issues/1230